src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,78 @@
  */
 
 public class Plane {
+    private String model;
+    private int countOfSeats;
+    private double price;
+    private int buildYear;
+    private String color;
 
+    private Plane(PlaneBuilder builder) {
+        this.model = builder.model;
+        this.countOfSeats = builder.countOfSeats;
+        this.price = builder.price;
+        this.buildYear = builder.buildYear;
+        this.color = builder.color;
```

Comment:
do we need to explicitly use `this` keyword in this case?

Reply:
done